src/enzo/RecordTotalActiveParticleCount.C

```diff
@@ -0,0 +1,62 @@
+/***********************************************************************
+/
+/  RECORD TOTAL NUMBER OF ACTIVE PARTICLES
+/
+/  written by: Ji-hoon Kim
+/  date:       October, 2009
+/  modified1:  December, 2011 by John Wise -- active particle counts
+/
+/  PURPOSE: 
+/
+************************************************************************/
+
+#ifdef USE_MPI
+#endif
+
+#include "preincludes.h"
+#include "ErrorExceptions.h"
+#include "macros_and_parameters.h"
+#include "typedefs.h"
+#include "global_data.h"
+#include "Fluxes.h"
+#include "GridList.h"
+#include "ExternalBoundary.h"
+#include "Grid.h"
+#include "Hierarchy.h"
+#include "TopGridData.h"
+#include "LevelHierarchy.h"
+#include "CosmologyParameters.h"
+#include "CommunicationUtilities.h"
+
+void RecordTotalActiveParticleCount(HierarchyEntry *Grids[], int NumberOfGrids,
```

Comment:
This method is declared in ActiveParticleInitialize, but isn't used. Is this functionality duplicated elsewhere?

Reply:
There is a function prototype in ActiveParticleInitialize.C alright. I suspect someone used it at some point there. I've now removed the prototype and it looks like right now the code is not called from anywhere. I think it's best to just leave this code here. It may be useful in the future.